contrib/abseil-cpp-cmake/CMakeLists.txt

```diff
@@ -17,3 +17,17 @@ get_target_property(FLAT_HASH_SET_INCLUDE_DIR absl::flat_hash_set INTERFACE_INCL
 target_include_directories (_abseil_swiss_tables SYSTEM BEFORE INTERFACE ${FLAT_HASH_SET_INCLUDE_DIR})
 
 add_library(ch_contrib::abseil_swiss_tables ALIAS _abseil_swiss_tables)
+
+set(ABSL_FORMAT_SRC
+    ${ABSL_ROOT_DIR}/absl/strings/internal/str_format/arg.cc
```

Comment:
This lib looks roughly modeled after the `str_format_internal` lib in contrib/abseil-cpp/absl/strings/CMakeLists.txt. Maybe add a comment a la `/// modeled after str_format_internal in native build system`